### PR TITLE
chore: change info to debug logging

### PR DESCRIPTION
**Pull request by Tomvbe**

### CODE REVIEW

File: ldi-orchestrator/ldio-connectors/ldio-http-in/src/main/java/be/vlaanderen/informatievlaanderen/ldes/ldio/LdioHttpIn.java [hunk 1]
```diff
@@ -43,4 +43,11 @@ public RouterFunction<ServerResponse> mapping() {
 				});
 	}
 
+	private void logIncomingRequest(ServerRequest request) {
+		var httpMethod = HttpMethod.POST.name();
+		var type = request.headers().contentType().map(MediaType::toString).orElse("(unknown)");
+		long contentLength = request.headers().contentLength().orElse(0L);
+		log.atDebug().log("%s /%s type: %s length: %s".formatted(httpMethod, endpoint, type, contentLength));
```
Comment (jobulcke) [inline ldi-orchestrator/ldio-connectors/ldio-http-in/src/main/java/be/vlaanderen/informatievlaanderen/ldes/ldio/LdioHttpIn.java:50]: Is there a reason why you would not use the built in formatting from the debugger in stead of the `String.formatted()`?

File: ldi-orchestrator/ldio-connectors/ldio-kafka/src/main/java/be/vlaanderen/informatievlaanderen/ldes/ldio/LdioKafkaIn.java [hunk 2]
```diff
@@ -30,8 +33,9 @@ public LdioKafkaIn(ComponentExecutor executor, LdiAdapter adapter, String defaul
 	@Override
 	public void onMessage(ConsumerRecord<String, String> data) {
 		final String contentType = determineContentType(data.headers());
-		getAdapter().apply(LdiAdapter.Content.of(data.value(), contentType))
-				.forEach(getExecutor()::transformLinkedData);
+		final var content = LdiAdapter.Content.of(data.value(), contentType);
+		log.debug("Incoming kafka message: %s".formatted(content));
```
Comment (jobulcke) [inline ldi-orchestrator/ldio-connectors/ldio-kafka/src/main/java/be/vlaanderen/informatievlaanderen/ldes/ldio/LdioKafkaIn.java:37]: Same remark here

**Review by jobulcke — APPROVED** [review]

**Review by jobulcke — CHANGES_REQUESTED** [review]
Fix code smell please

File: ldi-orchestrator/ldio-connectors/ldio-http-in/src/main/java/be/vlaanderen/informatievlaanderen/ldes/ldio/LdioHttpIn.java (hunk 1 above)
Comment (Tomvbe) [reply ldi-orchestrator/ldio-connectors/ldio-http-in/src/main/java/be/vlaanderen/informatievlaanderen/ldes/ldio/LdioHttpIn.java]: Nope, good remark. I started refactoring from from a String.formatted and didn't question it. I will change it.

**Review by jobulcke — APPROVED** [review]

**Pull request merged**